var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -143,6 +143,7 @@ def cmake_args(self):
 
         if '~shared' in self.spec:
             options.append('-DBUILD_SHARED_LIBS:BOOL=OFF')
+            options.append('-DGMXAPI:BOOL=OFF')
```

Comment:
Can we instead turn shared libs on?

Reply:
Shared libs are on by default. This change only applies if shared libs are disabled.